src/runtime-rs/src/shim-proto/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Do you need a full license file, rather than using an SPDX identifier?

Reply:
The `shim-proto` crate is designed to be published on crate.io. So it contains seperate LICENSE file